web/cypress/plugins/index.js

```diff
@@ -19,4 +19,12 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    if (browser.family === 'chromium' && browser.name !== 'electron') {
+      return {
+        ...launchOptions,
+        args: [...launchOptions.args.filter((arg) => arg !== '--disable-gpu'), '--ignore-gpu-blacklist'],
```

Comment:
So we always want to use GPU's for some reason?

Reply:
Judging by the error message I was getting in Chromium launched from Cypress, the map is rendered using WebGL, for which the GPU is required.

Reply:
Would be nice to add this explanation as a comment to the code :)

Reply:
Done.